envi/qt/html.py

```diff
@@ -128,13 +128,17 @@
 
     var elem = document.getElementById(name);
 
-    var elemrect = elem.getBoundingClientRect();
-    if ( elemrect.top < 0 ) {
-        elem.scrollIntoView(true);
-    }
-
-    if ( elemrect.bottom > window.innerHeight ) {
-        elem.scrollIntoView(false);
+    if (elem != null) {
```

Comment:
this will provide more specific context, even if it doesn't solve the sequencing issue.  good call.